sio-digital-team/active-safe
# Hotspot integration

https://trello.com/c/8X79EAQa

**What Changed:**
- [ ] Added ACF Addon for hotspots
- [ ] Updated ACF settings json file with the hotspot for Sports Landing Page
- [ ] Replaced hardcoded hotspots with data coming from WP
- [ ] Updated template file, CSS, and JS to make hotspots more function and mobile responsive based on the changes

**How to test:**
- Sync the custom fields to make sure you're bringing in the new fields under `Custom Fields` on the dashboard
- Edit the Sports Landing Page and upload an image
- Add hotspots by clicking on `Add Hotspot` and select the area, add a title, description, and link.
- Save and View the page to see if hotspot shows up and is in the correct position
- Click on the hotspot to open up the popover
